_pages/how-we-work/acquisition-engagements-staffing.md

```diff
@@ -0,0 +1,45 @@
+---
+title: Acquisition Engagement Staffing
+tags:
+- procurements
+- acqusitions
+- engagements
+- consulting
+- staffing
+---
+
+So you’ve been staffed to a procurement project. Here are some common questions that may come up.
+
+## What does "working towards a procurement" mean?
+
+You may have seen language about “working towards a procurement” in one of our partner agreements. In practice, these projects look very similar to any other 18F project, especially at the beginning. They typically start with a discovery period called a [Path Analysis](https://github.com/18F/path-analysis) where we work with the partner to define the problem space and assess their digital readiness.
+
+The Path Analysis deliverable is usually a roadmap with recommended next steps. For projects working towards a procurement, the next steps might include:
+
+* Investigation of the partner’s procurement capabilities and appetite for agile procurement methods (e.g. smaller contracts that allow for incremental progress and help avoid vendor lock-in). 
+* Market research to discover products and/or contractors with relevant expertise.
+* Developing a procurement strategy and solicitation package. (Don’t worry! The procurement experts in our Acquisitions chapter take the lead on this work.)
+* User research and prototyping work to surface product, design, technical, security, or deployment challenges that a future contractor might face. 
+
+he user research and prototyping work serve several specific purposes in a procurement project. First, information gleaned from this work can be used to inform a potential procurement or other next step. Additionally, technical barriers can be identified and potentially removed during this phase, setting up contractors to be successful faster. And lastly, this work allows 18F to model good practices for the partner. For example, the 18F team might demonstrate good scrum hygiene, working “in the open,” and the value of cross-functional teams, all of which can help foster better government/industry partnerships.
```

Comment:
The "T" is missing from the first sentence of this paragraph.

Reply:
Fixed